department/department_app/models/department.py

```diff
@@ -0,0 +1,13 @@
+"""models"""
+from django.db import models
+
+
+class Department(models.Model):
+    """Department"""
+    department = models.CharField(max_length=25)
+    dumber_of_employees = models.IntegerField(null=True)
```

Comment:
**D**umber_of_employees? :) I would recommend removing this field and generate it automatically (as for average_salary).

Reply:
fixed